src/main/java/org/radarcns/phone/PhoneLocationProvider.java

```diff
@@ -46,10 +46,10 @@
     public static final String MINIMUM_BATTERY_LEVEL_KEY = PREFIX + PHONE_LOCATION_BATTERY_LEVEL_REDUCED;
     public static final String REDUCED_BATTERY_LEVEL_KEY = PREFIX + PHONE_LOCATION_BATTERY_LEVEL_MINIMUM;
 
-    private static final int LOCATION_GPS_INTERVAL_DEFAULT = 60*60; // seconds
-    private static final int LOCATION_GPS_INTERVAL_REDUCED_DEFAULT = 5 * LOCATION_GPS_INTERVAL_DEFAULT; // seconds
-    private static final int LOCATION_NETWORK_INTERVAL_DEFAULT = 10*60; // seconds
-    private static final int LOCATION_NETWORK_INTERVAL_REDUCED_DEFAULT = 5 * LOCATION_NETWORK_INTERVAL_DEFAULT; // seconds
+    private static final int LOCATION_GPS_INTERVAL_DEFAULT = 15*60; // seconds
+    private static final int LOCATION_GPS_INTERVAL_REDUCED_DEFAULT = 4 * LOCATION_GPS_INTERVAL_DEFAULT; // seconds
+    private static final int LOCATION_NETWORK_INTERVAL_DEFAULT = 5*60; // seconds
+    private static final int LOCATION_NETWORK_INTERVAL_REDUCED_DEFAULT = 4 * LOCATION_NETWORK_INTERVAL_DEFAULT; // seconds
```

Comment:
Were these new intervals tested ? Whats the impact on battery life ?

Reply:
Are you referring to automated tests? I don't think that we have test cases to cover these. I do not know about the impact on battery-life. We have to test it in the future. We haven't done any long term test to analyze this.